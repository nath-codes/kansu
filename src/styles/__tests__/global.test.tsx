import 'jest-styled-components'
import React from 'react'

import { renderWithTheme } from '../../components/__tests__/helpers'
import GlobalStyles from '../global'

describe('GlobalStyles', () => {
  it('renders correctly', () => {
    const tree = renderWithTheme(<GlobalStyles />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
