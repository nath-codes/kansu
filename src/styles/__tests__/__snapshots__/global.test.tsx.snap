// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GlobalStyles renders correctly 1`] = `null`;
